enhancements/monitoring/replace-prometheus-adapter-with-metrics-server.md

```diff
@@ -0,0 +1,226 @@
+---
+title: replace-prometheus-adapter-with-metrics-server
+authors:
+  - "@slashpai"
+reviewers:
+  - "@simonpasquier"
+  - "@dgrisonnet"
+  - "@openshift/openshift-team-monitoring"
+approvers: 
+  - "@simonpasquier"
+  - "@openshift/openshift-team-monitoring"
+api-approvers:
+  - "@deads2k"
+creation-date: 2023-06-20
+last-updated: 2023-06-26
+tracking-link:
+  - https://issues.redhat.com/browse/MON-3153
+---
+
+# Replace prometheus-adapter with metrics-server
+
+## Summary
+
+This proposal suggests replacing the [Prometheus Adapter](https://github.com/kubernetes-sigs/prometheus-adapter) currently used for implementing the resource [metrics API](https://github.com/kubernetes/metrics) in OpenShift with the [Metrics Server](https://github.com/kubernetes-sigs/metrics-server).
+
+## Motivation
+
+For vertical and horizontal pod autoscaling, a resource metrics API implementation is necessary. Currently, OpenShift relies on the Prometheus stack and Prometheus adapter managed by the Cluster Monitoring Operator (CMO).
+
+Reasons to consider to moving to Metrics Server are:
+
+1. **Upcoming Archival of prometheus-adapter:** The Prometheus Adapter project will soon be archived due to difficulties in maintaining the project, making it less desirable for long-term usage.
+
+1. **Simplified Monitoring Architecture:** By replacing Prometheus Adapter with Metrics Server, we can reduce the complexity and maintenance overhead of the monitoring system in OpenShift since `metrics-server` does not rely on Prometheus.
```

Comment:
```suggestion
1. **Simplified and more robust Monitoring architecture:** By replacing Prometheus Adapter with Metrics Server, we can reduce the complexity and maintenance overhead of the monitoring system. Since `metrics-server` does not rely on Prometheus, the resources metrics API would remain functional even when the Prometheus service is down.
```